### PR TITLE
Lowercase: Nil return when output is equal to input

### LINKED ISSUE #1871: Lowercase: Should ignore input that's only numerical
https://duckduckgo.com/?q=lowercase+1235&t=ffab&ia=answer

We should ignore input that only has numbers. We should probably not show a result if the output === input actually?

------
IA Page:  http://duck.co/ia/view/lowercase

### PR DESCRIPTION
**Pull request by altern8tif**
Fixes #1871.

### CODE REVIEW

File: t/Lowercase.t [hunk 1]
```diff
@@ -11,14 +11,8 @@ zci is_cached   => 1;
...
-    'lowercase foo' => test_zci(
-        'foo',
-        structured_answer => {
-            input     => ['foo'],
-            operation => 'Lowercase',
-            result    => 'foo'
-        },
-    ),
+    'lowercase foo' => undef,
+    'lowercase 123' => undef,
```
Comment (GuiltyDolphin) [inline t/Lowercase.t]: @altern8tif Maybe add a case with mixed lowercase and numeric input? And a case with mixed uppercase and numeric input.

Comment (altern8tif) [inline t/Lowercase.t]: :+1:

File: lib/DDG/Goodie/Lowercase.pm [hunk 2]
```diff
@@ -25,12 +25,16 @@ handle remainder => sub {
 
     my $lower = lc $input;
 
-    return $lower,
-      structured_answer => {
-        input     => [html_enc($input)],
-        operation => 'Lowercase',
-        result    => html_enc($lower)
-      };
+    if ($lower eq $input) {
```
Comment (mintsoft) [inline lib/DDG/Goodie/Lowercase.pm]: I'd probably write this as 
```perl
return if ($lower eq $input);
return $lower,
    structured_answer => {
        input     => [html_enc($input)],
        operation => 'Lowercase',
        result    => html_enc($lower)
    };
```

**Comment by moollaza** [comment]
@altern8tif Thanks! This LGTM :+1:

**Pull request merged**